cfgen/cfgen

```diff
@@ -95,6 +95,7 @@ pools:
         node: <str>  # optional; 'hostname' of the corresponding node
     data_units: <int>
     parity_units: <int>
+    spare_unit: <int>
```

Comment:
I think it should be 'spare_units: <int>'.

Reply:
Done. Thanks!